MapleServer2/Commands/Game/InfoCommands.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using MapleServer2.Commands.Core;
+using MapleServer2.Packets;
+using MapleServer2.Tools;
+using MapleServer2.Enums;
+
+namespace MapleServer2.Commands.Game
+{
+    public class InfoCommands : InGameCommand
+    {
+        public InfoCommands()
+        {
+            Aliases = new[]
+{
```

Comment:
Formatting issue or gituhub bug

Reply:
Don't know what happen with this, it always open like this, even if fix the formatting. Going to try redo the file

Reply:
Don't know what happen with this, it always open like this, even if fix the formatting. Going to try redo the file